src/common/constants.ts

```diff
@@ -29,6 +29,19 @@ export const frameworkMap: Record<Framework['id'], Framework['name']> = {
   16: 'fxrecord',
 };
 
+export const frameworks: Framework[] = [{ id: 1, name: 'talos' },
+{ id: 2, name: 'build_metrics' },
+{ id: 4, name: 'awsy' },
+{ id: 6, name: 'platform_microbench' },
+{ id: 10, name: 'raptor' },
+{ id: 11, name: 'js-bench' },
+{ id: 12, name: 'devtools' },
+{ id: 13, name: 'browsertime' },
+{ id: 15, name: 'mozperftest' },
+{ id: 16, name: 'fxrecord' },
```

Comment:
[question] id is not hard coded right ?

Reply:
Each framework has an ID that comes from the backend, yes. I assume, though I'm not sure, that this also correlates to an ID in the database?

This is the API endpoint: https://treeherder.mozilla.org/api/performance/framework/